linkerd/proxy/http/src/lib.rs

```diff
@@ -34,12 +34,12 @@ pub use http::{
     uri, Method, Request, Response, StatusCode,
 };
 pub use http_body::Body;
+pub use hyper_util::rt::tokio::TokioExecutor;
 pub use linkerd_http_box::{BoxBody, BoxRequest, BoxResponse, EraseResponse};
 pub use linkerd_http_classify as classify;
 pub use linkerd_http_detect::{
     DetectMetrics, DetectMetricsFamilies, DetectParams, Detection, NewDetect,
 };
-pub use linkerd_http_executor::TracingExecutor;
```

Comment:
i've reexported this from `linkerd-proxy-http`, as we did for the previous executor.

i took the time to update callers to use the new name, rather than doing this under an alias. vim macros made this a straight-forward bit of work.